### DISCUSSION TITLE
谷歌网盘无法挂载

### DISCUSSION THREAD

**Discussion by Meeeeet6** [AlistGo/alist#3008] [Q&A]
谷歌网盘无法挂载，获取不到刷新令牌，求救，错误如下图所示
![image](https://user-images.githubusercontent.com/74483113/211843174-5f0dd031-7e12-4bc8-9e59-0e125b64c926.png)
![image](https://user-images.githubusercontent.com/74483113/211843191-fc248749-b713-43f3-9a90-9608c54dcf52.png)

**Comment by meowmeow123221** [comment]
same problem with you

**Reply by anwen-anyi** [reply]
https://www.bilibili.com/video/BV18v4y1W7vo/
